docs/faq.md

```diff
@@ -21,7 +21,7 @@ the documentation sources.
 
 ### How much does Engine cost?
 
-Docker Engine is 100% free. It is open source, so you can use it without paying.
+The open source Docker Engine is available for free. Commercial support is also available for the Docker Engine as a standalone product or as part of the Docker Datacenter subscription. Pricing details are available at [https://www.docker.com/pricing](www.docker.com/pricing)
```

Comment:
These docs are published at [docs.docker.com](https://docs.docker.com/engine/faq/), which contains documentation for all Docker projects (both open-source and commercial).
I'm sure there are lots of people trying to figure out the cost and support level for Docker projects, and I think people might search for that in our docs.